src/components/ShadowedCard/ShadowedCard.js

```diff
@@ -145,27 +151,32 @@ class ShadowedCard extends React.Component<Props, State> {
     const readyToRenderShadow = finishedAnimating || !isAnimated;
     const animatedContentOpacity = !cardHeight ? 0 : 1;
     const contentOpacity = isAnimated ? animatedContentOpacity : 1;
+    const isDarkTheme = currentTheme === DARK_THEME;
+
     return (
       <AnimatedCardOutter
         style={{ ...wrapperStyle, transform: [{ scaleY: isAnimated ? scaleValue : 1 }] }}
         ref={ref => { this.cardOutterRef = ref; }}
         disabled={disabled}
       >
-        {!noShadow && !!(cardHeight && cardWidth) && readyToRenderShadow && currentTheme !== DARK_THEME &&
-        <Shadow
-          heightAndroid={cardHeight}
-          heightIOS={cardHeight}
-          widthIOS={cardWidth}
-          widthAndroid={cardWidth}
-          useSVGShadow
-          wrapperStyle={{
-            position: 'absolute',
-            top: -(SHADOW_LENGTH / 2),
-            left: -(SHADOW_LENGTH / 2),
-            opacity: disabled ? 0.4 : 0.8,
-          }}
-          shadowRadius={(borderRadius || 6) - 2}
-        />}
+        {!noShadow && !!(cardHeight && cardWidth) && readyToRenderShadow && (forceShadow || !isDarkTheme) &&
```

Comment:
Just thinking about difference of `forceShadow` and `noShadow` – both can "force" either use or not to use shadow 🤔